src/amt/models/amt_models.ts

```diff
@@ -94,7 +94,7 @@ export interface AMT_EnvironmentDetectionSettingData extends CIM_SettingData<AMT
 export interface AMT_SystemDefencePolicy extends CIM_ManagedElement<AMT_SystemDefencePolicy> {
   PolicyName?: string
   PolicyPrecedence?: number
-  AntiSpoofingSupport?: number
+  antiSpoofingSupport?: number
```

Comment:
This should remain uppercase A, additionally we should rename `AMT_SystemDefencePolicy` to `AMT_SystemDefensePolicy` to match the SDK. https://software.intel.com/sites/manageability/AMT_Implementation_and_Reference_Guide/default.htm?turl=HTMLDocuments%2FWS-Management_Class_Reference%2FAMT_SystemDefensePolicy.htm

Reply:
done